src/transformers/models/longformer/modeling_longformer.py

```diff
@@ -425,7 +425,7 @@ def _compute_global_attention_mask(input_ids, sep_token_id, before_sep_token=Tru
 
 
 # Copied from transformers.models.roberta.modeling_roberta.create_position_ids_from_input_ids
```

Comment:
Better to delete the `Copied from ...` here... Longformer is far away from being a decoder in `EncoderDecoder`